subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[jvm_test_suite_plugin]]
+= The JVM Test Suite Plugin
+
+The JVM Test Suite plugin (plugin id: `jvm-test-suite`) allows for defining multiple groups of automated tests in a Java project.  These tests can be compiled once and run multiple times against various different target environments (for example, on different JVMs).
+
+This plugin can be used to define a group of Integration Tests, which might run much longer than unit tests and have different environmental requirements, and ought to be run against all the different JVMs which will be supported in production.
+
+[[sec:jvm_test_suite_usage]]
+== Usage
+
+This plugin is applied automatically by the `java` plugin, but can be explicitly applied if desired:
```

Comment:
Hmm, I'm not sure it makes any sense to apply the jvm-test-suite plugin without anything else. I think the behavior right now might be broken without the java plugin applied.

Reply:
I understand, but didn't mean to imply that the plugin could be applied standalone. Let me try to reword.